SPLASH ACTIVITY

edwin-pixel-1 opened the pull request:
Implementation of Splash activity with 3.0 seconds of delay and Logo
animation.

edwin-pixel-1 committed:
ADDED SPLASH ACTIVITY

Implementation of Splash activity with 3.0 seconds of delay and Logo
animation.

lucaslabs commented on app/src/main/java/com/github/globant/githubsubscribers/commons/utils/Constants.java:
```diff
@@ -0,0 +1,24 @@
+    public static final boolean DEBUG_LOGS = true;
+
+    public static final long SPLASH_DELAY = 3000;
+    public final static class SPLASH_LOGO_ANIMATION {
```
Rename to a Camel Case class name.

edwin-pixel-1 committed:
CAMEL CASE CONSTANT CLASS

- Fixed to Camel Case name of SplashLogoAnimation static class in
  Constants class

lucaslabs commented:
LG

daniellemariani commented:
LG
